shell/common/rasterizer.cc

```diff
@@ -81,7 +81,8 @@ void Rasterizer::Setup(std::unique_ptr<Surface> surface) {
 #if !defined(OS_FUCHSIA)
   // TODO(sanjayc77): https://github.com/flutter/flutter/issues/53179. Add
   // support for raster thread merger for Fuchsia.
-  if (surface_->GetExternalViewEmbedder()) {
+  if (surface_->GetExternalViewEmbedder() &&
+      surface_->GetExternalViewEmbedder()->SupportDynamicThreadMerging()) {
```

Comment:
I think Android's `SupportDynamicThreadMerging` should return `true` as well otherwise it would break hybrid composition on Android

Reply:
This issue will be surfaced in presubmits once the scenario app starts running.  For now, I don't think any test would fail until they engine is rolled into to the framework.

Reply:
Since android hasn't implemented dynamic thread merging yet. It needs to be false. We will enable it when implements thread merging on android.

Reply:
https://github.com/flutter/engine/blob/master/shell/platform/android/external_view_embedder/external_view_embedder.cc#L288 . This is already a dependency, it's just that the thread configuration is static.

Reply:
Ah I see, done